Functions/src/main/java/aws/devopsguru/partner/servicenow/ServiceNowConnector.java

```diff
@@ -25,8 +32,7 @@ public class ServiceNowConnector {
     public static final String NEW_LINE = System.lineSeparator();
 
     // Configure your Environment variables in AWS Lambda.
-    public static String username = System.getenv("USER_NAME");
```

Comment:
Are you removing the option to pass user name and password ?. Have you confirmed this with Blog writers ?

Reply:
Yes we need to remove secret name and password, as main purpose of using secret manager is to protect our credentials. Will discuss it with blog writers.